### PR TITLE
NozomiGuardian Integration Same As Closed PR #7112

### PR DESCRIPTION
**Pull request by ayman-m**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [x ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ x] Tests
- [ x] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by ayman-m** [comment]
@yaakovi I hope this one works, I can see only one commit pushed now

File: Packs/NozomiNetworks/TestPlaybooks/playbook-NozomiGuardian_Test.yml [hunk 1]
```diff
@@ -0,0 +1,228 @@
+id: NozomiGuardian-Test
+name: NozomiGuardian-Test
+version: -1
+fromversion: 4.5.0
```
Comment (yaakovi) [inline Packs/NozomiNetworks/TestPlaybooks/playbook-NozomiGuardian_Test.yml:4]: integration has `fromversion: 5.0.0`
```suggestion
fromversion: 5.0.0
```

**Comment by yaakovi** [comment]
@kirbles19 please review integration YML and README

File: Packs/NozomiNetworks/Integrations/NozomiGuardian/NozomiGuardian.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  dockerimage: demisto/python3:3.8.3.8715
+  feed: false
+  isfetch: false
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  subtype: python3
+  type: python
+fromversion: 5.0.0
```
Comment (yaakovi) [inline Packs/NozomiNetworks/Integrations/NozomiGuardian/NozomiGuardian.yml:83]:
```suggestion
fromversion: 5.0.0
tests:
- NozomiGuardian-Test
```

**Comment by kirbles19** [comment]
@yaakovi Done.

**Comment by yaakovi** [comment]
will fix the build errors in the base branch

**Pull request merged**